server/JsDbg.Gdb/JsDbg.py

```diff
@@ -102,17 +102,12 @@ def DebuggerQuery(tag, command):
 
 
 def FormatType(symbol_type):
-        pointer_depth = 0
         t = symbol_type.strip_typedefs()
-        while t.code == gdb.TYPE_CODE_PTR or t.code == gdb.TYPE_CODE_ARRAY:
-            pointer_depth = pointer_depth + 1
-            t = t.target()
-
         if (t.code == gdb.TYPE_CODE_FUNC):
```

Comment:
Does this work with function pointers, or arrays of/pointers to function pointers?

I know that when I was first writing this, function pointers were a pain to round-trip, and if you don't strip off the pointers/arrays you won't know whether it is a function pointer

Reply:
OK, that should be fixed now